### PR TITLE
Remove tool injection for runs

### PR DESCRIPTION
**Pull request by rick-a-lane-ii**
Initial phased deployment to remove the (unused and deprecated) Tool Injection use from Runs.

### CODE REVIEW

File: tests/e2e/tenant_test.go [hunk 1]
```diff
@@ -275,43 +274,3 @@ func TestTenantNamespaceUpdate(t *testing.T) {
 		}
 	})
 }
-
-func TestTenantToolInjection(t *testing.T) {
```
Comment (impl) [inline tests/e2e/tenant_test.go:279]: Probably should leave this one until we remove the Knative side too.

File: pkg/operator/app/pipeline.go [hunk 2]
```diff
@@ -100,11 +100,9 @@ func ConfigurePipelineParts(ctx context.Context, p *PipelineParts) error {
 		return err
 	}
 
-	if p.Deps.ToolInjectionCheckout.Satisfied() {
-		p.Pipeline.SetWorkspace(tektonv1beta1.PipelineWorkspaceDeclaration{
-			Name: ToolsWorkspaceName,
-		})
-	}
+	p.Pipeline.SetWorkspace(tektonv1beta1.PipelineWorkspaceDeclaration{
```
Comment (impl) [inline pkg/operator/app/pipeline.go:103]: I'm almost positive that you can remove all this workspace-specific stuff too, including anything that might mount it into the containers.

**Review by impl — APPROVED** [review]
LGTM

File: pkg/operator/app/pipeline.go (hunk 2 above)
Comment (rick-a-lane-ii) [reply pkg/operator/app/pipeline.go:103]: Yeah we can, but I was going to reuse it. (although we don't specifically need to...)
But the code is currently written/functional using the Tekton workspace for the tool mounts.

**Pull request merged**